src/prelude/internals.mo

```diff
@@ -397,38 +397,62 @@ let @ic00 = actor "aaaaa-aa" :
     } -> async ()
  };
 
-func @ic00_create_canister() : shared {
-      settings : ?@ManagementCanister.canister_settings
-    } -> async { canister_id : Principal } {
-  @ic00.create_canister
-};
-
-func @ic00_install_code() : shared {
-    mode : { #install; #reinstall; #upgrade };
-    canister_id : Principal;
-    wasm_module : @ManagementCanister.wasm_module;
-    arg : Blob;
-  } -> async () {
-  @ic00.install_code
+func @install_actor_helper(
```

Comment:
The core of the installation code, expressed as a more readable helper rather than hand-constructed IR (now that we can use async*/await* for decent abstraction)